website/CASE_STUDY_TEMPLATE.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Case study title" 
+name: "Company/project name"
+icon: /images/logos/powered-by/company_name.png
+hasNav: true
```

Comment:
@bullet03 can you check if this parameter is required?

Reply:
added to the ADD_LOGO.md